Framework/DataObjects/src/EventWorkspaceMRU.cpp

```diff
@@ -141,18 +141,18 @@ void EventWorkspaceMRU::insertE(size_t thread_num, EType data, const EventList *
  * @param index :: index to delete.
  */
 void EventWorkspaceMRU::deleteIndex(const EventList *index) {
```

Comment:
Could the cases with non-empty `m_bufferedDataE` and non-empty `m_bufferedDataY` be treated separately?